core/src/test/groovy/com/novoda/gradle/release/TestConfiguration.groovy

```diff
@@ -73,9 +81,21 @@ class TestBintrayRelease {
         Node packagingNode = nodes[5]
         assertThat(packagingNode.name().localPart).isEqualTo "packaging"
         assertThat(packagingNode.value()[0]).isEqualTo "aar"
+    }
+
+    @Test
+    public void testBintrayUploadTask() {
+        Task task = project.getTasks().findByPath(":bintrayUpload")
+        assertThat(task).isNotNull()
+
+        BintrayUploadTask uploadTask = task as BintrayUploadTask
+        uploadTask.user = "user"
+        uploadTask.apiKey = "apiKey"
 
-        // uploadTask verification here
+        uploadTask.repoName = "maven"
+        uploadTask.packageName = "test"
+        uploadTask.executeWithoutThrowingTaskFailure()
 
-        project.buildDir.deleteDir()
+        assertThat(uploadTask.dryRun).isFalse()
```

Comment:
This should assert for isTrue. However, the task is not configured (see comment above).

@devisnik  @hzsweers  Any ideas how to improve that?